src/core/oauth2-authorize.js

```diff
@@ -34,14 +34,44 @@ export default function authorize ( auth, authActions, errActions, configs ) {
       + "&state=" + encodeURIComponent(state)
       + "&client_id=" + encodeURIComponent(clientId)
 
-  // pass action authorizeOauth2 and authentication data through window
-  // to authorize with oauth2
-  win.swaggerUIRedirectOauth2 = {
-    auth: auth,
-    state: state,
-    callback: authActions.preAuthorizeOauth2,
-    errCb: errActions.newAuthErr
-  }
+  if (flow === "application") {
+    fetch(schema.get("tokenUrl"), {
```

Comment:
It is done. It is now the form as you requested.
Please let me know your thoughts..
THX!